README.md

```diff
@@ -12,6 +12,17 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 2. Run `codeclimate engines:enable eslint`. This command both installs the engine and enables it in your `.codeclimate.yml` file.
 3. You're ready to analyze! Browse into your project's folder and run `codeclimate analyze`.
 
+### Configuration
+
+#### `ignore_warnings`
+By default, ESLint rules at the warning level will be registered as errors. You can ignore warnings by setting this flag:
```

Comment:
To match Code Climate parlance a bit more, I might go with something like:

"By default, this engine will emit both ESLint errors and warnings as Code Climate issues. If you prefer, you can ignore warning-level violations by setting the `ignore_warnings` configuration option:"

What do you think?